client/src/document/index.scss

```diff
@@ -54,6 +52,18 @@
     }
   }
 
+  h1 {
+    @include mdn.uify();
+    &::after {
+      text-decoration-color: var(--category-color);
+    }
+  }
+
+  /* reduce the top margin of the interactive example heading */
+  h2:first-of-type {
+    margin-top: 2rem;
+  }
```

Comment:
This also affects the "Prerequisistes" section on http://localhost:3000/en-US/docs/Learn/HTML:

<img width="795" alt="image" src="https://user-images.githubusercontent.com/495429/164081324-27bcc49e-5247-458b-95ff-6e701b2246ac.png">

I would tend to remove this exception. Or, if there really should be less spacing before interactive examples, to add a CSS class to that heading via `EmbedInteractiveExamp.ejs`:

https://github.com/mdn/yari/blob/1c28a240f988803c175e62d3c28acc5cc055a7a5/kumascript/macros/EmbedInteractiveExample.ejs#L51

Reply:
I actually did the second but, there is a Cheerio script that runs on build and it removed the class sadly.

Reply:
I think this is OK for now in the context of the larger changes that form part of this. One can always go back and clean this up as part of the larger SASS/CSS refactor that I believe Florian has in mind.